docker-compose.yml

```diff
@@ -11,8 +11,9 @@ version: "2"
 services:
     db:
         image: mongo:3.2.6
-
-    server:
+        ports:
+            - "27017:27017" # Change port if needed
```

Comment:
I like being able to connect to mongo, like the change below, this is purely general dx, not related to reaching the goal described in #16936